gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/ApplicationAlertServiceImpl.java

```diff
@@ -490,4 +477,22 @@ private UpdateAlertTriggerEntity convert(AlertTriggerEntity trigger) {
 
         return updating;
     }
+
+    private List<Filter> combineFilters(String applicationId, List<Filter> filtersToAdd) {
+        List<Filter> filters = new ArrayList<>();
+        filters.add(StringCondition.equals("application", applicationId).build());
+        if (filtersToAdd != null) {
+            filters.addAll(filtersToAdd);
+        }
+
+        return filters;
+    }
+
+    private List<Notification> combineNotifications(List<Notification>... notificationsToAdd) {
+        return Arrays.stream(notificationsToAdd).filter(Objects::nonNull).flatMap(Collection::stream).collect(Collectors.toList());
+    }
+
+    private void removeNotificationsByNotifier(Trigger alert, String notifier) {
+        alert.getNotifications().removeIf(n -> notifier.equals(n.getType()));
+    }
```

Comment:
This method feels weird here because we are updating the argument. I would either create a `removeNotificationsByNotifier` in the `Trigger` class or just inline the content as it's used only once.

Reply:
As discussed together, i'll put it inline